### PR TITLE
Release Sealed Secrets 0.18.2

### PR DESCRIPTION
**Pull request by josvazg**
**Description of the change**

Update the Release Notes so we can trigger a new release `v0.18.2`

**Benefits**

New release!

### CODE REVIEW

**Review by alemorcuq — COMMENTED** [review]
I don't think we should include chart related notes in this changelog.

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,9 +1,132 @@
...
 [![](https://img.shields.io/github/release/bitnami-labs/sealed-secrets.svg)](https://github.com/bitnami-labs/sealed-secrets/releases/latest)
 
+## v0.18.2
+
+### Changelog
+
+- Replace ioutil with io or os ([#895](https://github.com/bitnami-labs/sealed-secrets/pull/895))
+- Remove CLI global variables and refactor flag handling ([#901](https://github.com/bitnami-labs/sealed-secrets/pull/901) & [#920](https://github.com/bitnami-labs/sealed-secrets/pull/920))
+- Upgrade Go version, dependencies and tooling ([#904](https://github.com/bitnami-labs/sealed-secrets/pull/904) & [#905](https://github.com/bitnami-labs/sealed-secrets/pull/905))
+- Allow skipping ClusterRole creation ([#906](https://github.com/bitnami-labs/sealed-secrets/pull/906)
```
Comment (alemorcuq) [inline RELEASE-NOTES.md:128]: This is also a chart related change.

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,9 +1,131 @@
...
+  - [v0.4.0](#v040)
+  - [v0.3.1](#v031)
+  - [v0.3.0](#v030)
+  - [v0.2.1](#v021)
+  - [v0.2.0](#v020)
+  - [v0.1.0](#v010)
+  - [v0.0.1](#v001)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
```
Comment (alemorcuq) [inline RELEASE-NOTES.md:114]: Not sure about how or why this was generated...

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,9 +1,131 @@
...
 [![](https://img.shields.io/github/release/bitnami-labs/sealed-secrets.svg)](https://github.com/bitnami-labs/sealed-secrets/releases/latest)
 
+## v0.18.2
+
+### Changelog
+
+- Replace ioutil with io or os ([#895](https://github.com/bitnami-labs/sealed-secrets/pull/895))
+- Remove CLI global variables and refactor flag handling ([#901](https://github.com/bitnami-labs/sealed-secrets/pull/901) & [#920](https://github.com/bitnami-labs/sealed-secrets/pull/920))
+- Upgrade Go version, dependencies and tooling ([#904](https://github.com/bitnami-labs/sealed-secrets/pull/904) & [#905](https://github.com/bitnami-labs/sealed-secrets/pull/905))
+
```
Comment (alemorcuq) [inline RELEASE-NOTES.md:128]: LGTM

**Review by alemorcuq — COMMENTED** [review]
The PR description has nothing to do with the contents, which one is correct?

**Comment by josvazg** [comment]
> The PR description has nothing to do with the contents, which one is correct?

Fixed

**Review by alemorcuq — APPROVED** [review]

**Pull request merged**